slackclient/_client.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, token):
         self.token = token
         self.server = Server(self.token, False)
 
+    def get_user_agent(self):
```

Comment:
Thinking about it, I'm not sure I need to expose this method, as it's only used by the request class to construct the header string for a request